### ISSUE TITLE
Deploying a compose on a vanilla cluster doesn't mount from local

### ISSUE THREAD

**Issue by jLopezbarb** [okteto/okteto#2792]
**Describe the bug**
If I have a compose file like the one above my nginx service won't have the nginx config from my local:
``` docker-compose.yml
services:
  api:
    image: nginx
    volumes:
      - nginx.conf:/etc/nginx/nginx.conf
```

**To Reproduce**

1. Create the docker-compose from above
2. Run `okteto context` on a vanilla cluster
3. Run `okteto deploy`
4. Check that the container doesn't have the file mounted from local

**Expected behavior**
I want that my application starts without problems

**Desktop (please complete the following information):**

- OS: macOS
- Version 2.4.beta-1

**Additional context**
We could give tips to the user on how to create the service on a vanilla cluster (probably a blog) and refer the users to that blog

**Comment by pistocop** [comment]
Hi @jLopezbarb, I'm facing a similar issue. I have seen that the fix of this issue is planned for the 2.6 version, is there any quick fix I can make meanwhile?
many thanks

**Comment by jLopezbarb** [comment]
Yes @pistocop, you can create a `Dockerfile` including the volumes like this:
``` Dockerfile
FROM $SERVICE_IMAGE
COPY $LOCALPATH $REMOTEPATH
```
And using the new image + removing/commenting them on the compose should do the trick:
``` yaml
services:
  api:
    image: $NEW_IMAGE
```
